### DISCUSSION TITLE
New plugin

### DISCUSSION THREAD

**Discussion by volucris1** [NvChad/NvChad#1110] [Ideas]
Found this plugin: https://github.com/ghillb/cybu.nvim
What do you think? I like use it instead BufferLineCycle
![Plugin preview](https://user-images.githubusercontent.com/35503959/169406683-6fb0c4dd-2083-4b9b-87b2-3928da81d472.gif)

**Comment by siduck** [comment]
@volucris1 its more about user preferability, you can add it yourself too!
